cypress/integration/organizations/integration-organization.spec.js

```diff
@@ -0,0 +1,99 @@
+import TopMenu from '../../support/fragments/topMenu';
+import permissions from '../../support/dictionary/permissions';
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import users from '../../support/fragments/users/users';
+import newOrganization from '../../support/fragments/organizations/newOrganization';
+import Organizations from '../../support/fragments/organizations/organizations';
+import InteractorsTools from '../../support/utils/interactorsTools';
+
+
+describe('ui-organizations: EDI convention in Organization Integration', () => {
+  let userId = '';
+  const organization = {
+    ...newOrganization.defaultUiOrganizations,
+    accounts: [
+      {
+        accountNo: `${getRandomPostfix()}`,
+        accountStatus: 'Active',
+        acqUnitIds: [],
+        appSystemNo: '',
+        description: 'Main library account',
+        libraryCode: 'COB',
+        libraryEdiCode: `${getRandomPostfix()}`,
+        name: 'Accout1',
+        notes: '',
+        paymentMethod: 'Physical Check',
+      },
+      {
```

Comment:
this objects seems the same except a couple of values.
you can create 1 default obj -> copy 2 times -> set new values
This is the better way because anyone can reuse it in other tests

Reply:
Done